lib/plugins/helper/open_graph.js

```diff
@@ -84,7 +84,13 @@ function openGraphHelper(options = {}) {
   }
 
   if (language) {
-    result += og('og:locale', language);
+    if (language.length === 5) {
+      const territory = language.slice(-2);
+
+      language = language.replace('-', '_').replace(territory, territory.toUpperCase());
```

Comment:
If the user specify `fo_fo` that results will be `FO_FO`.

Reply:
So, `toUpperCase` must be applied only to the last two characters.

Reply:
> If the user specify `fo_fo` that results will be `FO_FO`.

It will be `FO_fo`, `String.replace()` only replace the first occurrence.

I'm fixing it.

Reply:
Fixed.